README.md

```diff
@@ -1034,6 +1034,7 @@ vars:
 | `intermediate_prefixes` | the list of acceptable prefixes for your intermediate models | int_ |
 | `marts_prefixes` | the list of acceptable prefixes for your marts models | fct_, dim_ |
 | `other_prefixes` | the list of acceptable prefixes for your other models | rpt_ |
+| `prefer_model_folder_type_to_prefix` | override to prefer the use of the folder name to the prefix when determining model type | false |
```

Comment:
Hi! Could you update the description here to signal what the variable does, rather than what overriding it does?